package.json

```diff
@@ -265,6 +265,21 @@
                     ],
                     "default": null,
                     "description": "if `rust.workspace_mode` is enabled, only the specified package in the workspace will be analyzed."
+                },
+                "rust.features": {
+                    "type": "array",
+                    "default": [],
+                    "description": "A list of features to enable."
+                },
+                "rust.all_features": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Enable all features."
+                },
+                "rust.no_default_features": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Do not enable default features."
```

Comment:
For the descriptions could you s/features/Cargo features please to make clear what features we're talking about?

Reply:
Done, sorry it took me so long :p